src/miner_keys.erl

```diff
@@ -22,7 +25,7 @@ get_onboarding_key(Default) ->
         undefined -> Default;
         OnboardingKey ->
             case file:read_file(OnboardingKey) of
-                {ok, Bin} -> Bin;
+                {ok, Bin} -> string:trim(binary_to_list(Bin));
```

Comment:
Could you add a spec to this get_onboarding_key function to make it obvious what it returns?

Reply:
Certainly!

Reply:
Done.